lib/pf/constants/switch.pm

```diff
@@ -0,0 +1,55 @@
+package pf::constants::switch;
```

Comment:
We already have a module for constants for switch.
pf::Switch::constants

Reply:
I know but the package name is 'SNMP' and doesn't contain another package name that this could fit in

You prefer that I define another package in pf::Switch::constants and put this in there ?

Reply:
That package pf::Switch::constants is a bit messy.
Leave it where it is for we can start to clean it up later.
We can start to move the other constants from pf::Switch::constants later to the new package to keep things consistent.